### PR TITLE
allow pipeline to be setup in ocp cluster

### PR DESCRIPTION
**Pull request by harshad16**
## Related Issues

Related-to: https://github.com/AICoE/aicoe-ci/issues/58

## This introduces a breaking change

- [x] No

## This Pull Request implements

- added required resources to the inclusion list.

## Description

allow the pipeline to be set up in ocp cluster
Signed-off-by: Harshad Reddy Nalla <hnalla@redhat.com>

### CODE REVIEW

File: manifests/overlays/prod/configs/argo_cm/resource.inclusions [hunk 1]
```diff
@@ -6,6 +6,18 @@
   - https://api.thoth01.lab.upshift.rdu2.redhat.com:6443
   - https://api.thoth12.lab.upshift.rdu2.redhat.com:6443
   - https://api.ocp4.idh.dev.datahub.redhat.com:6443
+- apiGroups:
+  - '*'
```
Comment (HumairAK) [inline manifests/overlays/prod/configs/argo_cm/resource.inclusions:10]: the wildcard here might cause issues, can you specify the apigroups individually?

File: manifests/overlays/prod/configs/argo_cm/resource.inclusions [hunk 2]
```diff
@@ -323,4 +335,4 @@
   - Gateway
   - Sensor
   clusters:
-  - https://paas.stage.psi.redhat.com:443
\ No newline at end of file
+  - https://paas.stage.psi.redhat.com:443
```
[the remark this answers is not in the document]
Comment (harshad16) [reply manifests/overlays/prod/configs/argo_cm/resource.inclusions]: oh ok, will remove it :+1:

File: manifests/overlays/prod/configs/argo_cm/resource.inclusions (hunk 1 above)
Comment (harshad16) [reply manifests/overlays/prod/configs/argo_cm/resource.inclusions]: good point, updated the api groups.

**Review by HumairAK — APPROVED** [review]

**Pull request merged**